website/src/components/Chat/PluginsChooser.tsx

```diff
@@ -234,6 +236,16 @@ export const PluginsChooser = ({ plugins, setPlugins }: PluginsChooserProps) =>
               placeholder={t("plugin_url_placeholder")}
             />
             {selectedForEditPluginIndex !== null && <JsonCard>{plugins[selectedForEditPluginIndex]}</JsonCard>}
+            <Box>
+              <Text fontSize="md" fontWeight="bold" my={2}>
+                {t("plugin_repositories")}
+              </Text>
+              <Flex gap={2}>
+                <Button as={Link} href="https://www.getit.ai/gpt-plugins" target="_blank" rel="noreferrer">
```

Comment:
if you have you would have callback url for plugin installs you could use getit's callback mechanism: https://www.getit.ai/gpt-plugins/chat-integration

Reply:
We don't have this yet but sounds like a good idea .. could you maybe create a new issue for this?